content/docs/deployment/csmoperator/_index.md

```diff
@@ -15,17 +15,23 @@ Dell CSM Operator has been tested and qualified on Upstream Kubernetes and OpenS
 | -------------------------- | ------------------- |
 | 1.25, 1.26, 1.27           | 4.11, 4.12, 4.12 EUS |
 
+>NOTE:
+>- Authorization module is only supported on Kubernetes platforms.
+
+## Supported CSM Components
 
 The table below lists the driver and modules versions installable with the CSM Operator:
 
-| CSM Module         | PowerFlex  |  PowerScale  |  PowerStore |  PowerMax  |  Unity XT  |
-| ------------------ | ---------- | ------------ | ----------- | ---------- | ---------- |
-| CSI Driver         | ✔ 2.5.0+   | ✔ 2.5.0+    | ✔ 2.5.0+    | ✔ 2.6.0+   | ✔ 2.6.0+  |
-| CSM Authorization  | ✔ 1.4.0+   | ✔ 1.4.0+    | ✔ 1.4.0+    | ✔ 1.4.0+   | ✔ 1.4.0+  |
-| CSM Replication    | ✔ 1.3.0+   | ✔ 1.3.0+    | ❌          | ✔ 1.3.0+   | ❌        |
-| CSM Observability  | ✔ 1.3.0+   | ✔ v1.3.0+   | ❌          | ❌         | ❌         |
-| CSM Resiliency     | ✔ 1.6.0+   | ✔ v1.6.0+   |✔ v1.6.0+    | ❌         | ❌         |
+| CSI Driver         | Version | CSM Authorization | CSM Replication | CSM Observability | CSM Resiliency |
+| ------------------ |---------|-------------------|-----------------|-------------------|----------------|
+| CSI PowerScale     | 2.7.0   | 1.6.0             | 1.5.0           | 1.5.0             | 1.6.0          |
```

Comment:
Why remove the checkmarks and crosses ?
I prefer to make it clear and I apply the same syntax as here : https://dell.github.io/csm-docs/docs/#csm-modules-support-matrix-for-dell-csi-drivers

Reply:
Sure, I'll add the similar checkmarks for better readability.